src/components/pdf-outline-sidebar/expand.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"/></svg>
```

Comment:
This file is in the wrong directory.

Reply:
I think that having a directory with a single file is silly. Components usually have 1-2 images in addition so I prefer having images and styles in the component root.

If you prefer `assets`, I will move. Perhaps, I will move for consistency.